docs/src/documentation/03-components/07-interaction/stepper/03-accessibility.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Accessibility
+redirect_from:
+  - /components/stepper/accessibility/
+---
+
+## Accessibility
+
+### Stepper
+
+The Stepper component has been designed with accessibility in mind. It can be used with keyboard navigation and includes properties that enhance the experience for users of assistive technologies.
+
+Although the `ariaLabel` and `ariaLive` props are optional for the Stepper (StepperStateless) component itself, it is recommended to fill them in to ensure that the component can be perceived by assistive technologies. This ensures that the Stepper (StepperStateless) component is accessible.
+
+Also, it is highly recommended to fill the `titleDecrement` and `titleIncrement` props in to provide a description of the decrement and increment buttons for screen readers.
+
+#### Example
+
+The content of children component is text, so it is read by screen reader.
+
+```jsx
+<Stepper
+  step={1}
+  minValue={0}
+  minValue={10}
+  ariaLabel="Passengers number"
+  titleDecrement="Remove a passenger"
+  titleIncrement="Add a passenger"
+  ariaLive="assertive"
+/>
+```
+
+The screen reader will announce stepper title (`Passengers number`) and buttons title (`Add a passenger`, `Remove a passenger`) once they are focused by screen reader.
```

Comment:
Is it stepper title? I thought it's title of the input? From "stepper title" I would assume it's the title for the whole component, including the buttons and displayed text.

Will the screen reader first announce "Passengers number" and only then "Remove a passenger" and "Add a passenger"? It won't be "Remove a passenger", then "Passengers number" and then "Add a passenger"?

Reply:
Hm, but maybe it should be `value title` instead of `input title`? Because we are referring to the number with `value` name in the docs.